python/mxnet/_ctypes/ndarray.py

```diff
@@ -97,6 +121,11 @@ def generic_ndarray_function(*args, **kwargs):
         for i in args:
             if isinstance(i, NDArrayBase):
                 ndargs.append(i)
+            elif num_args.value == 1:
```

Comment:
This isn't valid. It breaks mx.nd.arange()
I think you should fix this by falling back on minpy side